resources/assets/lib/components/comment.coffee

```diff
@@ -211,6 +210,26 @@ export class Comment extends React.PureComponent
               label: osu.trans('comments.load_replies') if @children.length == 0
               ref: @loadMoreRef
 
+  
+  renderBlocked: =>
+    blockClass = classWithModifiers 'comment', @props.modifiers, top: @props.depth == 0
+    mainClass = classWithModifiers 'comment__main', 'deleted'
+
+    div className: blockClass,
+      div className: mainClass,
+        span 
+          className: if @props.depth > 0 then 'comment__avatar' else ''
+          style:
+            height: 'auto'
+
+        div className: 'comment__container',
+          div className: 'comment__main blocked comment__message osu-md osu-md--comment',
```

Comment:
`osu-md osu-md--comment` should be on the `p` element with `osu-md__paragraph`.
`comment__main blocked` shouldn't be here.